src/dparse/ast.d

```diff
@@ -800,9 +804,10 @@ final class AsmStatement : BaseNode
 {
     override void accept(ASTVisitor visitor) const
     {
-        mixin (visitIfNotNull!asmInstructions);
+        mixin (visitIfNotNull!(asmInstructions, gccAsmInstructions));
```

Comment:
Looks like the `functionAttributes` were missing in the old version of this code. Not sure if that was an intentional omission.

Reply:
No idea but I'll ad them to be consistent with the other AST nodes.